nsot/models/network.py

```diff
@@ -588,3 +615,21 @@ def to_dict(self):
             'state': self.state,
             'attributes': self.get_attributes(),
         }
+
+
+# Signals
+def refresh_assignment_interface_networks(sender, instance, **kwargs):
+    """This signal fires each time a Network object is saved. Upon save,
+    the signal iterates through all the child networks of the network
+    being saved and cleans the addresses and networks assigned to the
+    interfaces (if any) to which these child networks have been assigned."""
```

Comment:
This docstring describes _what_ is happening with the code here, but _why_ are we doing this? What are we accomplishing/fixing when we call `clean_addresses()`?

Reply:
Makes sense!